src/Ledger/Epoch.lagda

```diff
@@ -156,9 +146,83 @@ toRwdAddr x = record { net = NetworkId ; stake = x }
 getStakeCred : TxOut → Maybe Credential
 getStakeCred (a , _ , _ , _) = stakeCred a
 
-open RwdAddr using (stake)
 open GovActionState using (returnAddr)
 \end{code}
+\begin{NoConway}
+\Cref{fig:functions:createRUpd} defines
+the function \AgdaFunction{createRUpd}
+which creates a \AgdaRecord{RewardUpdate},
+i.e.\ the net flow of Ada due to paying out rewards
+after an epoch.
+Relevant quantities are:
+\begin{itemize}
+  \item \AgdaArgument{prevPp}: Previous protocol parameters,
+    which correspond to the parameters during the epoch for which we are creating rewards.
+  \item \AgdaFunction{Δr₁}: Ada taken out of the reserves for paying rewards,
+    as determined by the \AgdaField{monetaryExpansion} protocol parameter.
+  \item \AgdaFunction{rewardPot}: Total amount of coin available for rewards this epoch, as
+described in section 6.4 of \textcite{shelley-delegation-design}.
+  \item \AgdaFunction{feeSS}: The fee pot, containing the transaction fees from the epoch.
+    We use the fee pot from the snapshot taken at the epoch boundary.
+    Note that fees are not explicitly removed from any account:
+    the fees come from transactions paying them and are accounted for whenever transactions are processed.
+  \item \AgdaFunction{Δt₁}: The proportion of the reward pot that will move to the treasury,
+    as determined by the \AgdaField{treasuryCut} protocol parameter.
+    The remaining pot is called the \AgdaFunction{R},
+    just as in section 6.5 of \textcite{shelley-delegation-design}.
+  \item \AgdaFunction{pstakego}:
+    Stake distribution used for calculating the rewards.
+    This is the oldest stake distribution snapshot, labeled ``go''.
+  \item \AgdaFunction{rs}: The calculated rewards.
+    As given by \AgdaFunction{maxPool}, each pool can receive a maximal amount,
+    determined by its performance.
+  \item \AgdaFunction{Δr₂}: The difference between the maximal amount of rewards
+    that could have been paid out if pools had been optimal,
+    and the actual rewards paid out.
+    This difference is returned to the reserves.
+  \item \AgdaFunction{÷₀}: Division operator that returns zero when the denominator is zero.
+\end{itemize}
+
+\begin{figure*}[h]
+\begin{code}
+ActiveSlotCoeff : ℚ
+ActiveSlotCoeff = (1 / 20)
+\end{code}
+\begin{code}[hide]
+  {{nonZero}}
+\end{code}
+\begin{code}
+createRUpd : PParams → ℕ → BlocksMade → EpochState → Coin → RewardUpdate
+createRUpd prevPp slotsPerEpoch b es total
+  = ⟦ Δt₁ , 0 - Δr₁ + Δr₂ , 0 - feeSS , rs ⟧
+  where
+    reserves = es .EpochState.acnt .Acnt.reserves
+    pstakego  = es .EpochState.ss .Snapshots.go
+    feeSS    = es .EpochState.ss .Snapshots.feeSS
+    stake = pstakego .Snapshot.stake
+    delegs  = pstakego .Snapshot.delegations
+    poolParams = pstakego .Snapshot.poolParameters
```

Comment:
```suggestion
    reserves = es .EpochState.acnt .Acnt.reserves
    pstakego = es .EpochState.ss .Snapshots.go
    feeSS = es .EpochState.ss .Snapshots.feeSS
    stake = pstakego .Snapshot.stake
    delegs = pstakego .Snapshot.delegations
    poolParams = pstakego .Snapshot.poolParameters
```

Reply:
Unfortunately, making this change runs into error #753 .

However, it appears that by adding one empty newline after the function `createRUpd`, I can align these equal signs. Trying to align the other equal signs in the same function runs into the same error again, but at least these equal signs are aligned.